posts/2021/07/introducing-code-generator-v020.md

```diff
@@ -0,0 +1,60 @@
+
+## Deep Learning As a Routine
+
+In deep learning applications, neural networks are typically accompanied by the codebase to preprocess the input and output data, visualize the results, define proper training and evaluation pipelines, and more. A significant part of this supporting code consists of reusable components, like data loaders, training loops, logging, and tracking. Therefore, deep learning practitioners usually organize their boilerplate codebases into collections of reusable components to speed up the development.
+
+PyTorch-Ignite is one such practical solution, a high-level library from the PyTorch ecosystem for training neural networks designed to simplify workflow development while maintaining maximum control, flexibility, and reproducibility. PyTorch-Ignite feels like a natural extension to PyTorch.
+
+## Ignite Your Training Pipelines
+
+Ignite's Code Generator is an open-source tool developed to boost your training pipeline's scripts, carefully designed by PyTorch-Ignite's contributors to promote Ignite's best practices. The application has a user-friendly and intuitive web interface, simple enough for day-to-day use, and it is an excellent choice for fast configured PyTorch training script prototyping.
+
+In this release, we are using a new application development stack to enhance the user experience tenfold. For UI, we will proceed with the JavaScript stack, and the PyTorch framework and the PyTorch-Ignite library will remain as is for elaborating template code.
+
+### Getting Started
+
+The best way to "ignite your training pipeline" is to visit Code Generator's [homepage](https://code-generator.pytorch-ignite.ai/) and select your task's template by clicking on the "Getting Started" button.
+
+You can choose a template from a list of templates located on the left in the **Templates** tab. The app will start to render the template with the preconfigured default settings. You will see all the generated files with the rendered code on the right in different tabs as in a regular IDE. The current state of the configurations is reflected partially in the config YAML file.
+
+Currently, we offer four customizable templates for widely used deep learning tasks: Vision classification and segmentation, Text classification, and DCGAN.
+
+Start adjusting the code in the template by visiting different tabs on the left side:
+
+- **Training**: To turn on Distributed training
+- **Handlers**: To set up Checkpointing, Termination on NaNs, Early Stopping, etc
+- **Loggers**: To configure Logging
+
+Once you choose the appropriate settings, press the "Download" or "Open in Colab" button on the top to export generated code as a zip archive or a notebook, and follow any given additional steps. The `requirements.txt` file contains all the required dependencies, and the `README` contains all the necessary information for launching the script. The resulting archive contains both these files and the config file bundled together.
+
```

Comment:
This point is kind of the end of the "story" of how Code Generator v0.2.0 helps the user. It would be nice to end with a sentence like "You're now ready to add in your data and model!". Or really any kind of "off to the races sentence", it's a bit more exciting than ending on "here are some files".